docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -189,6 +189,12 @@ Note: Actually, Gitea supports only SMTP with STARTTLS.
 - `ENABLED`: Enable this to run cron tasks periodically.
 - `RUN_AT_START`: Enable this to run cron tasks at start time.
 
+### Cron - Cleanup old repository archives (`cron.archive_cleanup`)
+
+- `RUN_AT_START`: Enable this to run cron tasks at start time.
+- `SCHEDULE`: Cron syntax for scheduling update mirrors, e.g. `@every 1h`.
```

Comment:
it should be `Cron syntax for scheduling repository archive cleanup...`

Reply:
Oops, sorry. Fixed and squash-rebased against current master.